cassandra/README.md

```diff
@@ -3,34 +3,26 @@
 This is based on [riptano's jepsen](https://github.com/riptano/jepsen/tree/cassandra/cassandra).
 
 ## Current status
-- Support Apache Cassandra 3.11.x
+- Supports Apache Cassandra 3.11.x
 - Support `collections.map-test`, `collections.set-test`, `batch-test`, `counter-test`(only add) and `lwt-test`
   - Removed `lww-test` and `mv-test`
 
 ## How to test
-### Start the Docker Container
 
-- Add the following line to the Docker file of a node
-  - Because C* needs Java 8
-
-```diff
---- a/docker/node/Dockerfile-ubuntu
-+++ b/docker/node/Dockerfile-ubuntu
-@@ -7,4 +7,5 @@ RUN apt-get install -y openssh-server \
-     curl faketime iproute2 iptables iputils-ping libzip4 \
-     logrotate man man-db net-tools ntpdate psmisc python rsyslog \
-     sudo unzip vim wget apt-transport-https \
-+    && apt-get update && apt-get install -y openjdk-8-jre \
-     && apt-get remove -y --purge --auto-remove systemd
-```
+### Docker settings
+
+You will probably need to increase the amount of memory that you make available to Docker in order to run these tests with Docker. We have had success using 8GB of memory and 2GB of swap. More, if you can spare it, is probably better.
 
-- Run docker
+### Start the Docker Container
```

Comment:
Can you add a comment about starting docker?
We need the latest files from the docker folder in Jepsen.

Reply:
Thank you. I can do that, but do you not think it may just be better to merge upstream/jepsen into this repository? I suppose there have not been any changes in any of the other directories so should be fine?

Reply:
If you can merge, it's great.

Reply:
Cool. Let me try and see if any problems arise.